ci-operator/step-registry/assisted/tools/multi-arch/build/assisted-tools-multi-arch-build-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: assisted-tools-multi-arch-build
```

Comment:
same thing as with the job name. if it's doing building and publishing it's better naming it ``assisted-tools-multi-arch-build`` so people are aware it has side-effects